research/conditional/utils/model_utils.py

```diff
@@ -245,6 +248,46 @@ def get_expert_choice_args(args):
     }
 
 
+def get_expert_choice_with_parallel_ff_args(args):
```

Comment:
Please review carefully this function. It should split the compute between experts layer and parallel ff layer.